### PR TITLE
Adding lhotse recipe to prepare eval2000 data

### PR DESCRIPTION
**Pull request by GoVivace**

### CODE REVIEW

**Comment by desh2608** [comment]
My impression was that eval2000 is used as a test set for Switchboard? Would it make sense to integrate the preparation with the switchboard recipe or do people also just use it by itself?

**Comment by GoVivace** [comment]
That will be done later in icefall.

**Comment by s-mousmita** [comment]
> Thanks, looks good! I left a few comments. Also two more questions:
> 
> 1. Can you add a row in the table of recipes: https://github.com/lhotse-speech/lhotse/blame/master/docs/corpus.rst#L40
> 2. Do you think it makes sense to add text normalization option / function that is specific to this corpus? We typically don't have text norm utils in Lhotse, but since this is a known and established benchmark, maybe it'd be useful.


2. We have a separate script to do the normalization. I thought that will go to the icefall/egs/fisher_swbd/ASR/local/   ,  as I saw normalization script for fisher and swbd there.  Do you want us add it here as a method   ?

File: lhotse/recipes/eval2000.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    :param corpus_path: Path to global corpus                                                                                                
+    :param output_dir: Directory where the manifests should be written. Can be omitted to avoid writing.                                     
+    :param absolute_paths: Whether to return absolute or relative (to the corpus dir) paths for recordings.                                  
+    :return: A dict with manifests. The keys are: ``{'recordings', 'supervisions'}``.                                                        
+    """
+
+    corpus_dir = Path(corpus_dir)
+    output_dir = Path(output_dir)
+    output_dir.mkdir(parents=True, exist_ok=True)
+    print(corpus_dir)
```
[the remark this answers is not in the document]
Comment (s-mousmita) [reply lhotse/recipes/eval2000.py:38]: Sure will do the change.

**Comment by GoVivace** [comment]
For text normalization, we were thinking that the local folder inside icefall will be a good place, because there are a lot of user discretionary things are done there, and if someone wants to get the original transcriptions, they will have the option of skipping this step.

**Comment by pzelasko** [comment]
Thanks! Can you run `black` on the code? We'll merge then.

**Comment by s-mousmita** [comment]
> Thanks! Can you run `black` on the code? We'll merge then.

Done.  Thanks.

**Comment by pzelasko** [comment]
Oh I missed it. It seems the `black` tests are still failing. Please make sure you are using a compatible black version (`pip install -U black==22.3.0`)

**Comment by s-mousmita** [comment]
> Oh I missed it. It seems the `black` tests are still failing. Please make sure you are using a compatible black version (`pip install -U black==22.3.0`)

Sorry, I had an older version of black. Reformatted with 22.3. Please check now. 
Thanks.

**Comment by pzelasko** [comment]
Thanks!

**Pull request merged**